tmt/steps/provision/__init__.py

```diff
@@ -289,6 +290,14 @@ def _query_has_selinux(self, guest: 'Guest') -> Optional[bool]:
 
         return 'selinux' in output.stdout
 
+    def _query_is_superuser(self, guest: 'Guest') -> Optional[bool]:
+        output = self._execute(guest, Command('whoami'))
+
+        if output is None or output.stdout is None:
+            return None
+
+        return output.stdout.strip() == 'root'
```

Comment:
what about other superuser accounts? superuser is just a definition of permissions, root is just the most commonly used one, I can have any name user as a superuser

maybe checking for something specific like writing to /var/tmp/tmt? or whatever else it is we need superuser for

Reply:
Would you mind if I'd file an issue for this & find a better way later? I agree this would miss other `root`-like accounts, on the other hand, I suspect changing the test will require more work, this PR just moves the stuff around, more or less.

Reply:
Sure, I understand this might cause a lot more problems and this is likely necessary ASAP.